### DISCUSSION TITLE
Voice crack audio

### DISCUSSION THREAD

**Discussion by AwesomeJackify** [voicepaw/so-vits-svc-fork#478] [General]
In the output file, when parts of the audio has that voice crack sound effect, what is happening. 
I have tried changing the octave but it doesn't help much.
Is this to do with the source audio or with the setup?

**Comment by Mangio621** [comment]
Here's what I think is happening: 

1. The source audio has harmonies and vocal layers in the song. For example, some choruses/verses will have vocal layers which confuses and causes problems with inference.
2. either the model being used on inference doesn't have enough training on pitches that are on the source audio? For example, vocal range issues.
3. f0 algorithms and their failure cases...

Make sure that the source audio doesn't have harmonies. Make sure the source audio is within the model's vocal range (can be fixed with transposition). Use crepe as it is more pitch "accurate" and has less failure cases compared to other f0 algorithms. I found that crepe handles harmonies well unlike dio etc. Make sure the source audio is clean and doesn't have any other instrument in it.

**Reply by AwesomeJackify** [reply]
Thanks for the breakdown, I will investigate further
